code/modules/mob/living/combat.dm

```diff
@@ -115,24 +118,159 @@
+					i++
+				if(2)
+					visible_message("<span class='borange'>[attacker] and [defender] both picked [defender.rps_intent]. Stalemate!</span>")
+		if(attacker_wins == 0)
+			attacker_wins = 1
+		if(defender_wins == 0)
+			defender_wins = 1
+		attacker.HideUI("Rock Paper Scissors Cards")
+		defender.HideUI("Rock Paper Scissors Cards")
+		switch(attacker_wins > defender_wins)
+			if(1)
+				visible_message("<span class='borange'>[attacker] wins!</span>")
+				visible_message("<span class='borange'>[attacker_wins/defender_wins] percentage!</span>")
+				returner = attacker_wins/defender_wins
+
+				attacker.DisplayUI("RPS Winner Beg Cards")
+				defender.DisplayUI("RPS Loser Beg Cards")
+				sleep(30)
+				attacker.HideUI("RPS Winner Beg Cards")
+				defender.HideUI("RPS Loser Beg Cards")
+				winner_beg_stance = attacker.rps_mercy_or_more
+				loser_beg_stance = defender.rps_mercy_or_more
+			if(0)
+				visible_message("<span class='borange'>[defender] wins!</span>")
+				visible_message("<span class='borange'>[(defender_wins/attacker_wins) * -1] percentage!</span>")
+				returner = (defender_wins/attacker_wins) * -1
+				defender.DisplayUI("RPS Winner Beg Cards")
+				attacker.DisplayUI("RPS Loser Beg Cards")
+				sleep(30)
+				defender.HideUI("RPS Winner Beg Cards")
+				attacker.HideUI("RPS Loser Beg Cards")
+				winner_beg_stance = defender.rps_mercy_or_more
+				loser_beg_stance = attacker.rps_mercy_or_more
+		if(winner_beg_stance == loser_beg_stance)//There must be a better way, but code optimization is for LATER
+			if(winner_beg_stance == "mercy")
+				attacker.rps_in_combat = 0
+				defender.rps_in_combat = 0
+				attacker.anchored = 0
+				attacker.canmove = 1
+				defender.anchored = 0
+				defender.canmove = 1
+				return returner
+			if(winner_beg_stance == "more")
+				chance_mercy = 100
+		else
+			if((winner_beg_stance == "more") && (loser_beg_stance == "mercy"))
+				chance_mercy = 77
+			if((winner_beg_stance == "mercy") && (loser_beg_stance == "more"))
+				chance_mercy = 33
+		if(prob(chance_mercy))
+			if(b % 2)//Hopefully makes sure that the total round ammount is always odd
+				b = 2
+			else
+				b = 3
+		else
+			attacker.rps_in_combat = 0
+			defender.rps_in_combat = 0
+			attacker.anchored = 0
+			attacker.canmove = 1
+			defender.anchored = 0
+			defender.canmove = 1
+			return returner
+
+/mob/proc/rps_win_check(var/mob/living/attacker, var/mob/living/defender)
+	if((attacker.rps_intent=="rock" && defender.rps_intent=="scissors") || (attacker.rps_intent=="scissors" && defender.rps_intent=="paper") || (attacker.rps_intent=="paper" && defender.rps_intent=="rock"))
+		return 0
+	else if((defender.rps_intent=="rock" && attacker.rps_intent=="scissors") || (defender.rps_intent=="scissors" && attacker.rps_intent=="paper") || (defender.rps_intent=="paper" && attacker.rps_intent=="rock"))
+		return 1
+	else if(defender.rps_intent == attacker.rps_intent)
+		return 2
```

Comment:
Magic numbers ! Please use a define for these.

Reply:
I have no idea what you mean by that.

Reply:
> I have no idea what you mean by that.

Instead of using arbitrary numbers (the 1 and 2), define them with #define and call that instead.

Reply:
I'll use that, once I figure it out.